MobileBuy/buy/src/main/java/com/shopify/buy/model/Address.java

```diff
@@ -62,12 +62,29 @@
 
     private String zip;
 
+    @SerializedName("default")
+    private Boolean defaultAddress;
```

Comment:
I switched it to Boolean for the property, and boolean for the getter and setter with a default value if Boolean property = null.